src/_gettsim/transfers/rente.py

```diff
@@ -811,20 +962,27 @@ def ges_rente_wartezeit_5(
         See basic input variable :ref:`m_freiw_beitrag <m_freiw_beitrag>`.
     m_ersatzzeit
         See basic input variable :ref:`m_ersatzzeit <m_ersatzzeit>`.
+    ges_rente_params
+        See params documentation :ref:`ges_rente_params <ges_rente_params>`.
 
     Returns
     -------
-    Wartezeit in years.
+    Bool of fulfilling Wartezeit von 5 Jahren.
```

Comment:
```suggestion
    Fulfilled Wartezeit von 5 Jahren.
```
Same elsewhere; we have the type hints so we don't need to repeat them.

Reply:
changed this